src/script/components/Modals/GroupCreation/GroupCreationModal.tsx

```diff
@@ -227,6 +233,44 @@ const GroupCreationModal: React.FC<GroupCreationModalProps> = ({
           createNavigate(generateConversationUrl(conversation.qualifiedId))(event);
         }
       } catch (error) {
+        const NON_FEDERATING_BACKENDS = 409;
+
+        if (isAxiosError<NonFederatingBackendsData>(error) && error.response?.status === NON_FEDERATING_BACKENDS) {
+          const tempName = groupName;
+          setIsShown(false);
+          const backendString = error.response?.data!.non_federating_backends?.join(', and ');
+          const replaceBackends = replaceLink(
+            'https://support.wire.com/hc/en-us/articles/9357718008093-Backend',
```

Comment:
```suggestion
            'https://support.wire.com/hc/articles/9357718008093',
```

removing lang and title allow support for other languages